jax/experimental/host_callback.py

```diff
@@ -1959,43 +1958,41 @@ def barrier_wait(logging_name: Optional[str] = None):
       for this invocation. See `Debugging` in the module documentation.
   """
   logging_name = logging_name or ""
-  if logging.vlog_is_on(2):
-    logging.vlog(2, f"barrier_wait[{logging_name}]: start")
+  if logger.isEnabledFor(logging.DEBUG):
```

Comment:
There's no need for the `isEnabledFor` guard here because you fixed the string interpolation! So the debug call does no unnecessary work and we can just call it unconditionally.

Reply:
Log guard (sic) removed.